.drone.yml

```
@@ -200,7 +200,7 @@ steps:
       repo:
         - joomla/joomla-cms
       branch:
-        - 4.0-dev
+        - 4.1-dev
```

this has to be reverted as rips has not yet been setup for 4.1

```suggestion
        - 4.0-dev
```